app/views/handlers/relationship_collector.py

```diff
@@ -103,3 +103,24 @@ def _is_last_relationship(self):
         if self.relationship_router.get_next_location_url(self._current_location):
             return False
         return True
+
+    def _resolve_custom_page_title(self, page_title: str) -> str:
+        for_list = self.block.get("for_list")
+
+        list_item_index = (
+            self._questionnaire_store.list_store.list_item_index(
+                for_list, self.current_location.list_item_id
+            )
+            + 1
+        )
+
+        to_list_item_index = (
+            self._questionnaire_store.list_store.list_item_index(
+                for_list, self.current_location.to_list_item_id
+            )
+            + 1
+        )
+
+        return page_title.format(
+            list_item_index=list_item_index, to_list_item_index=to_list_item_index
+        )
```

Comment:
We should consider one common `_resolve_custom_page_title` method to remove the duplication.

Reply:
I've made this change in 067d303 however, I am not sure I prefer the single method approach. Having two separate methods seems to me clearer and more explicit, but I'm not too bothered either way. Let me know what you think